### DISCUSSION TITLE
Column mode in DBeaver Query

### DISCUSSION THREAD

**Discussion by angeloruggieridj** [dbeaver/dbeaver#19731] [Q&A] [answered]
Hi,
is it possible to add in next software releases column mode feature, like Notepad++, using Alt key (https://npp-user-manual.org/docs/editing/)?
It would be very helpful especially when you have to add a prefix on a table column in a SELECT statement.

GIF example: https://npp-user-manual.org/docs/images/columnMode.gif

I hope you could consider this advice!
Thanks

**Comment by ShadelessFox** [comment, marked as answer]
Hello @angeloruggieridj,

Have you tried using block selection? It can be toggled in the <kbd>Edit</kbd> &rArr; <kbd>Toggle Block Selection</kbd>, or by using shortcut <kbd>Alt+Shift+A</kbd>.

Please let me know if it solves your issue.

**Comment by angeloruggieridj** [comment]
Thanks for the advice. Toggle Block Selection is what I need!
